### PR TITLE
Development

### PR DESCRIPTION
**Pull request by deniscbrex**

### CODE REVIEW

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello guys!! 

It's great to see you submit your first project for review. I have check your code and viewed it across different browsers. Here are some points I have for you;

- Your HTML file contains some errors. You can detect and fix the errors by parsing your code through the [W3C markup validator](https://validator.w3.org/)
- The margin/padding around your main content is too much.  the frame and "up-next" sidebar seem to compress. Reduce the margin/padding around their container and your clone would improve.
- You should implement the "subscribe" section and comment section as well.

When you're done, request for another code review and add me "James" as the TSE so I can follow up on it.

Happy coding!

File: youtube-video/style.css [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+}
+.subscribe-container{
+  width: 34%;  
+}
+
+
+.comment-section{
+  display: flex;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
```
Comment (JamesOkunlade) [inline youtube-video/style.css]: Refactor line 252 and 253 into a single line. margin-bottom: 1.5em 0;

Comment (NeerNim) [reply youtube-video/style.css]: refactored the css and you mean margin: 1.5em 0; :)

**Review by JamesOkunlade — APPROVED** [review]
### Hello guys!!

Your project now renders well and good enough across multiple browsers. Your code still contains an avoidable error though. It seems you didn't parse it through the [W3C markup validator](https://validator.w3.org/) after all. 

One last thing you should do before merging to master; 
- Your img tag should always contain the alt attribute. Fix this and your code will be error free.

When you're done with this, merge to master, close pull request and fill the project completion form.

Here are your thumbs up :+1::+1: Good luck with your future project.

Happy coding! :sunglasses:

**Review by JamesOkunlade — COMMENTED** [review]
### Great work guys!!

Just before you merge to master, check line 103 in your HTML code and remove one of the double alt attributes you used.

When you're done, merge to master, close pull request, and fill the project completion form.

Good luck with your future projects :+1::+1:

**Pull request merged**